udata/migrations/2024-01-29-fix-reuse-and-dataset-with-private-None.py

```diff
@@ -0,0 +1,36 @@
+'''
+Some old dataset/reuse don't have a `private` field. It should be False by default.
+'''
+import logging
+
+from udata.models import Reuse, Dataset
+
+log = logging.getLogger(__name__)
+
+
+def migrate(db):
+    log.info('Processing Reuse.')
+
+    reuses = Reuse.objects().no_cache().timeout(False)
+    count = 0
+    for reuse in reuses:
+        if reuse.private == None: 
```

Comment:
Since it defaults to False, it doesn't show as `None` here.
We may need to manipulate collections directly as was done in https://github.com/opendatateam/udata/blob/master/udata/migrations/2023-02-08-rename-internal-dates.py

Reply:
Thanks!